### PR TITLE
Moves custom exceptions into exceptions package

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
This moves the custom exceptions into the `exception` package

### CODE REVIEW

**Comment by dcbriccetti** [comment]
Is there an advantage to organizing exceptions into a single package, rather than defining them closer to where they are used? A modern IDE makes it very easy to find all the Throwables an application defines, wherever they may be, if that is needed.

**Comment by shobhitagarwal1612** [comment]
I did define them earlier as inner classes but there already exists a package which contains Custom Exceptions. So, just to follow the existing coding pattern, I did the refactoring.

File: collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.odk.collect.android.exception;
```
Comment (nribeka) [inline collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java:15]: Maybe you need to add the license too here.

File: collect_app/src/main/java/org/odk/collect/android/exception/MultipleFoldersFoundException.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package org.odk.collect.android.exception;
```
Comment (nribeka) [inline collect_app/src/main/java/org/odk/collect/android/exception/MultipleFoldersFoundException.java:16]: And here too.

File: collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java (hunk 1 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java:15]: Done

File: collect_app/src/main/java/org/odk/collect/android/exception/MultipleFoldersFoundException.java (hunk 2 above)
Comment (shobhitagarwal1612) [reply collect_app/src/main/java/org/odk/collect/android/exception/MultipleFoldersFoundException.java:16]: Done

**Comment by lognaturel** [comment]
This change is fine by me but the licenses don't really make sense! I think what happened is that you copied it from another one of the exceptions in that package but the license should reflect who actually added the files (not Dobility/UW in the case of these new ones).

**Comment by shobhitagarwal1612** [comment]
Pushed the changes. Still got to learn a lot about the licenses :sweat_smile:

File: collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2017 Shobhit
+ *
+ * Originally developed by Dobility, Inc. (as part of SurveyCTO)
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java]: I disagree with this statement. 😬😆

Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/exception/BadUrlException.java]: (in other words, Dobility did not create that file)

**Comment by shobhitagarwal1612** [comment]
@lognaturel Is the license ok now? Sorry for the trouble

**Comment by lognaturel** [comment]
@shobhitagarwal1612 No worries. Can't wait to get rid of all of them opendatakit/governance#2

**Pull request merged**